xxhash.c

```diff
@@ -635,6 +711,36 @@ static U64 XXH_read64(const void* memPtr)
 
 #if defined(_MSC_VER)     /* Visual Studio */
 #  define XXH_swap64 _byteswap_uint64
+#if defined(_M_IX86)
```

Comment:
indent macros so that we can clearly understand that this is a subsection on `_MSC_VER`.